expo/tests/scheduler/test_manage_volunteer_opportunities.py

```diff
@@ -0,0 +1,274 @@
+from django.core.urlresolvers import reverse
+import nose.tools as nt
+from django_nose.tools import assert_redirects
+from unittest import TestCase
+from django.test import Client
+from tests.factories.gbe_factories import (
+    ProfileFactory,
+    RoomFactory
+)
+from scheduler.models import EventContainer
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    login_as,
+)
+from tests.contexts import (
+    StaffAreaContext,
+)
+
+
+class TestEventList(TestCase):
+    view_name = 'manage_opps'
+
+    def setUp(self):
+        self.client = Client()
+        self.user = ProfileFactory.create().user_object
+        self.privileged_profile = ProfileFactory()
+        self.privileged_user = self.privileged_profile.user_object
+        grant_privilege(self.privileged_user, 'Volunteer Coordinator')
+
+    def test_no_login_gives_error(self):
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["1"])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 302)
```

Comment:
Should have brought this up before merging, but we should now be using follow=True and testing to see where we get to. Just testing status code == 302 only tells us that we redirected, but doesn't actually test anything useful.